src/dc_interface.cpp

```diff
@@ -3,42 +3,25 @@
 MotorController::DcInterface::DcInterface() {
 }
 
-bool MotorController::DcInterface::getEnable() const {
-    return enable;
-}
-
-void MotorController::DcInterface::setEnable(const bool state) {
-    enable = state;
-    enablePin.set(state);
-}
-MotorController::MotorInterface::Direction MotorController::DcInterface::getDirection() const {
-    return direction;
-}
-
-void MotorController::DcInterface::setDirection(const MotorController::MotorInterface::Direction newDirection) {
-    direction = newDirection;
-}
-
 int16_t MotorController::DcInterface::getSpeed() const {
     return speed;
 }
 
 void MotorController::DcInterface::setSpeed(const int16_t newSpeed) {

```

Comment:
You generally never want to print anything in a function. If you leave out the prints, this bit of logic can be simplified as well.

Reply:
printout is there to help remember the implementation of PWM